src/nvim/CMakeLists.txt

```diff
@@ -326,6 +307,34 @@ if(WIN32)
       -DDST=${PROJECT_BINARY_DIR}/windows_runtime_deps
       -P ${PROJECT_SOURCE_DIR}/cmake/WindowsDllCopy.cmake)
   add_dependencies(nvim_runtime_deps nvim_dll_deps)
+
+  add_custom_target(external_blobs
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${PROJECT_BINARY_DIR}/windows_runtime_deps/platforms
+
+    COMMAND ${CMAKE_COMMAND} -E copy "${DEPS_PREFIX}/bin/ca-bundle.crt"       ${PROJECT_BINARY_DIR}/windows_runtime_deps/
```

Comment:
> WindowsDllCopy.cmake was running dumpbin on the .exe files, which seems useless (and wrong

The dumpbin step is used to determine the DLLs that need to be shipped with each .exe i.e. part of the list you added below by hand. The advantage was that this would work with any binary in path.

swiss.exe seems to be self contained (does it have any DLL dependencies?). win32yank does need a couple DLLs.

Concerning nvim-qt, if it is never used during the tests or build, then there is no need to put it in DEPS_INSTALL and mix it with the other files there, only to separate it later with a long list of copy commands. This is different from diff or win32yank that should be available for test runs. In fact you could run that GetBinaryDep install recipe right into the nvim target install dir (problem with that is third-party does not know the install path).  Since technically this is not a build dependency, maybe the recipe could just be in the main cmake project as a post-install target.